### PR TITLE
[ci][full-ci] Use updated fedora-38 squish image for tests

### PR DESCRIPTION
**Pull request by saw-jan**
Updated squish image to have necessary libs for `master` branch in https://github.com/owncloud-ci/squish/pull/42

Bumped squish image to `owncloudci/squish:fedora-38-...`

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>GUI-tests-ocis</strong> https://drone.owncloud.com/owncloud/client/15303/6/1

:boom: The GUI tests failed.

GUI Logs: https://cache.owncloud.com/public/owncloud/client/15303/ocis/guiReportUpload/index.html

<details>

Server Logs: https://cache.owncloud.com/public/owncloud/client/15303/ocis/guiReportUpload/serverlog.log

Screenshots:
- [Open_log_dialog_with_Ctrl+l_keys_combination.png](https://cache.owncloud.com/public/owncloud/client/15303/ocis/guiReportUpload/screenshots/Open_log_dialog_with_Ctrl+l_keys_combination.png)

</details>

**Comment by ownclouders** [comment]
Results for <strong>GUI-tests-oc10</strong> https://drone.owncloud.com/owncloud/client/15304/6/1

:boom: The GUI tests failed.

GUI Logs: https://cache.owncloud.com/public/owncloud/client/15304/oc10/guiReportUpload/index.html

<details>

Server Logs: https://cache.owncloud.com/public/owncloud/client/15304/oc10/guiReportUpload/serverlog.log

Screenshots:
- [Open_log_dialog_with_Ctrl+l_keys_combination.png](https://cache.owncloud.com/public/owncloud/client/15304/oc10/guiReportUpload/screenshots/Open_log_dialog_with_Ctrl+l_keys_combination.png)

</details>

File: test/gui/tst_checkAlltabs/test.feature [hunk 1]
```diff
@@ -14,7 +14,7 @@ Feature: Visually check all tabs
             | Settings     |
             | QuitOwncloud |
 
-
+    @skip
```
Comment (saw-jan) [inline test/gui/tst_checkAlltabs/test.feature:17]: Squish image `squish:fedora-38` has a problem showing desktop-ui with the drone agent os (i.e. Ubuntu 20.04).

The background is all black:

Open_log_dialog_with_Ctrl+l_keys_combination

**Comment by saw-jan** [comment]
@TheOneRing @fmoc Can we merge this. I have skipped one failed test (until the issue gets resolved) and others are working fine (we can get green drone ci)

**Pull request merged**